<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制图像包含函数一(蓝)函数二(红):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>固定图像(隐藏/显示):</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">ont: 14pt &quot;MV Boli&quot;;
background-color: rgb(255, 250, 244);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="Line" name="linefen1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>16</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="linefen3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>20</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>59</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>函数式子:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;线&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>y=x^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=ln(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=sin(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=cos(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=sqrt(x)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>59</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>图像捕捉:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;线&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>y=x^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=ln(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=sin(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=cos(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y=sqrt(x)</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>61</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>向上左移:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>减:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>向上加:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>64</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>向下右移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>加:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox3">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;楷体&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>向下减:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox4">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="linefen2_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>21</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>函数一</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>函数二</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>0</y>
       <width>51</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_41">
        <property name="text">
         <string>向上左移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>减:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxF1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_43">
        <property name="text">
         <string>向上加:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxF2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget21">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>51</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_44">
        <property name="text">
         <string>向下右移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_45">
        <property name="text">
         <string>加:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxF3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_46">
        <property name="text">
         <string>向下减:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxF4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>隐藏</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>171</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">ont: 14pt &quot;MV Boli&quot;;
background-color: rgb(255, 250, 244);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>551</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>164</width>
       <height>18</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>graphicsView1</tabstop>
  <tabstop>comboBox1</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>comboBox2</tabstop>
  <tabstop>spinBox1</tabstop>
  <tabstop>spinBox2</tabstop>
  <tabstop>spinBox3</tabstop>
  <tabstop>spinBox4</tabstop>
  <tabstop>spinBoxF1</tabstop>
  <tabstop>spinBoxF2</tabstop>
  <tabstop>spinBoxF3</tabstop>
  <tabstop>spinBoxF4</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>graphicsView2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
